### PR TITLE
Document ome-xml unit classes

### PR DESCRIPTION
**Pull request by rleigh-dundee**
- Link to current name for deprecated units
- Document Unit and Quantity classes
- Document concrete Quantity types

### CODE REVIEW

File: components/ome-xml/src/ome/units/quantity/Temperature.java [hunk 1]
```diff
@@ -28,24 +28,29 @@
 import ome.units.unit.Unit;
 
 /**
- * A wrapper for the Temperature class from the units implementation.
+ * Length quantity.
```
Comment (dgault) [inline components/ome-xml/src/ome/units/quantity/Temperature.java]: Temperature quantity

**Comment by dgault** [comment]
Documentation all reads well and deprecated units are all correct.

The only one nitpicky thing is that they all use 'create an' which reads a bit off for certain types eg Length 'Create an Length' -> 'Create a Length'

**Comment by rleigh-codelibre** [comment]
@dgault Should all be fixed now.

**Comment by dgault** [comment]
Everything looks good with the changes. PR is ready to merge.

**Pull request merged**